frontends/mit-learn/src/page-components/MetaTags/MetaTags.tsx

```diff
@@ -71,7 +71,7 @@ const canonicalPathname = (pathname: string) => {
 }
 
 const SITE_NAME = APP_SETTINGS.SITE_NAME
-const DEFAULT_OG_IMAGE = `${window.origin}/static/images/mit-and-logo.jpg`
```

Comment:
I've left the old image in place for now so social previews still work between release / prerender cache clear.. We should delete it in a followup PR. (If not sooner, it will be deleted when we switch to nextjs branch.)